testing/requirements.txt

```diff
@@ -0,0 +1,11 @@
+# Dependencies installed (with !pip install) in example notebooks are listed below.
+# Note: These dependencies are installed during the `test_all_notebooks` and 
+# `test_changed_notebooks` github workflows. The dependencies of notebooks excluded 
+# from testing (see `EXCLUDE_NOTEBOOKS` in `testing/test_notebooks.py`) are not 
+# included here.
+alibi==0.6.2
```

Comment:
Any reason to pin this?

Reply:
Yea I think latest stable (via `alibi<1.0.0`) is good, also might catch some bugs between libraries.

Reply:
Ok sounds good!